Makefile

```diff
@@ -105,18 +105,21 @@ build: ## By default we should build in release mode
 
 .PHONY: build-no-std
 build-no-std: ## Build without the standard library
-	${BUILD_KERNEL_ERRORS} cargo build --no-default-features --target wasm32-unknown-unknown --workspace --lib
+	${BUILD_KERNEL_ERRORS} cargo build --no-default-features --target wasm32-unknown-unknown --workspace --lib --exclude miden-proving-service --features tx-prover
```

Comment:
I could be wrong but I think this change is not necessary because of this target just builds libs

Reply:
I checked this, without excluding that crate the command is not working because of some of its dependencies.
In my understanding, if we use `--workspace` we are first creating and compiling the dependency graph, therefore causing errors because some deps aren't no-std compatible (e.g. pingora).
It is not happening in `bench-tx` because the deps in the TOML are no-std compatible, and we only use `std` modules in the `main.rs`.

Reply:
Ah, I see, makes sense. I was taking `bench-tx` as an example but didn't realize that dependencies were `no_std` compatible.